### ISSUE TITLE
DFO data list folder structure is changed

### ISSUE THREAD

**Issue by webgisdeveloper** [Global-Flood-Assessment/MoMProduction#52]
https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2023
```
Name | Type | Size
-- | -- | --
001 | Directory | 0
002 | Directory | 0
360 | Directory | 0
361 | Directory | 0
362 | Directory | 0
363 | Directory | 0
364 | Directory | 0
365 | Directory | 0

Name	Type	Size
[001](https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2023/001)	Directory	0
[002](https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2023/002)	Directory	0
[360](https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2023/360)	Directory	0
[361](https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2023/361)	Directory	0
[362](https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2023/362)	Directory	0
[363](https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2023/363)	Directory	0
[364](https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2023/364)	Directory	0
[365](https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2023/365)	Directory	0
```

**Comment by webgisdeveloper** [comment]
https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2022
```


Name | Type | Size
-- | -- | --
001 | Directory | 0
002 | Directory | 0
360 | Directory | 0
361 | Directory | 0
362 | Directory | 0
363 | Directory | 0
364 | Directory | 0
365 | Directory | 0

Name	Type	Size
[001](https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2022/001)	Directory	0
[002](https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2022/002)	Directory	0
[360](https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2022/360)	Directory	0
[361](https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2022/361)	Directory	0
[362](https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2022/362)	Directory	0
[363](https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2022/363)	Directory	0
[364](https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2022/364)	Directory	0
[365](https://nrt4.modaps.eosdis.nasa.gov/api/v2/content/archives/allData/61/MCDWD_L3_NRT/2022/365)	Directory	0
```

**Comment by webgisdeveloper** [comment]
generate_procesing_list in DFO_tool need to be updated.

**Comment by webgisdeveloper** [comment]
datelist = generate_procesing_list()
> {'001': '20230101', '002': '20230102', '360': '20231226', '361': '20231227', '362': '20231228', '363': '20231229', '364': '20231230', '365': '20231231'}

**Comment by webgisdeveloper** [comment]
after the fix, the late date will is skipped
> {'001': '20230101', '002': '20230102'}

**Comment by webgisdeveloper** [comment]
make this issue as fixed now, anything new will be listed under #54
